styles/global.scss

```diff
@@ -426,6 +426,13 @@ a.has-text-grey {
 .k-cursor-pointer {
   cursor: pointer !important;
 }
+
+.k-grey {
+  @include ktheme() {
+    background-color: theme('k-grey');
+  }
+}
```

Comment:
please don't, ref #7354

Reply:
I add a new class `k-text-grey` to avoid using this class

Reply:
please change to `has-text-k-grey` to keep consistancy with Bulma css helpers

Reply:
kk